Sources/Tools/MaxConvert/hsControlConverter.cpp

```diff
@@ -127,7 +127,12 @@ void hsControlConverter::DeInit()
 class KRStatus : public KeyReduceStatus
 {
     void Init(int total) override { }
+
+#if MAX_VERSION_MAJOR >= 23
+    KeyReduceResult Progress(int p) override { return KEYREDUCE_CONTINUE; }
+#else
```

Comment:
It might be better for us to define `KeyReduceResult` in `MaxCompat.h` for older Maxes to reduce clutter.